samples/pom.xml

```diff
@@ -34,6 +34,11 @@
 
   <name>Apache Apex Malhar (incubating) Samples</name>
 
+  <properties>
+    <semver.plugin.skip>true</semver.plugin.skip>
```

Comment:
No issue with the plugin management part. But I do not see the point of repeating the plugin like this:
```
      <plugin>
        <groupId>com.github.siom79.japicmp</groupId>
        <artifactId>japicmp-maven-plugin</artifactId>
      </plugin>
```
in every module. Instead, I would prefer to see this in the malhar parent pom only and use the property that turns it off for those modules that should be exempt. That's how the deploy plugin is setup already.